go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -2166,7 +2170,9 @@ func (fbo *folderBranchOps) identifyOnce(
 	defer fbo.identifyLock.Unlock()
 
 	ei := tlfhandle.GetExtendedIdentify(ctx)
-	if fbo.identifyDone && !ei.Behavior.AlwaysRunIdentify() {
+	if (fbo.identifyDone && !ei.Behavior.AlwaysRunIdentify()) ||
+		(fbo.identifyDoneWithWarning &&
+			ei.Behavior.WarningInsteadOfErrorOnBrokenTracks()) {
```

Comment:
So even if `ei.Behavior.AlwaysRunIdentify()` is true, we could still skip. Though I guess AlwaysRunIdentify and WarningInsteadOfErrorOnBrokenTracks are mutually exclusive.

Reply:
Good point, I'll put it all in a big `AlwaysRunIdentify` statement.